src/components/Button/Button.js

```diff
@@ -1,22 +1,24 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled, {ThemeProvider} from 'styled-components';
+import sharedComponent from '../../sharedComponent';
 import icons from '../Icon/icons';
 import SubmitButton from './SubmitButton';
 import {secondaryTheme} from '../../theme';
 
 const ButtonImpl = styled.button`
-  font-size: ${({ theme }) => theme.button.fontSize};
+  font-size: ${({ mods }) => mods.small ? '0.7em' : '0.8em'};
   text-decoration: none;
-  font-weight: ${({ theme }) => theme.button.fontWeight};
-  font-family: ${({ theme }) => theme.button.fontFamily};
-  text-transform: ${({ theme }) => theme.button.textTransform};
+  font-weight: 700;
+  font-family: ${({ theme }) => theme.fonts.brand};
+  text-transform: uppercase;
 
   background: ${({ theme }) => theme.colors.primary};
   border: 1px solid ${({ theme }) => theme.colors.primary};
   color: ${({ theme }) => theme.colors.white};
   border-radius: 3em;
-  padding: ${({ theme }) => theme.button.padding};
+  /* subtracting 1 from the design system height due to the border */
```

Comment:
Comment says we are subtracting 1, but the code statically sets the values, unclear what the logic should be?

Reply:
The design system is a reference to the 'spec' at https://dev.bandwidth.com/design-system, not any value present in the code. I just wanted to leave a note about why it's 12px instead of 14px in case anyone saw it and wanted to 'fix' it. Though the comment itself is not that clear. It's copied over from the old theme files.